INTEGRATION.md

```diff
@@ -35,6 +35,39 @@ Integration here has 2 different meanings.
 
 2. One way to handle the protocols is to use the [HTTP gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/). This document describes how to determine which gateway to use.
 
+### $CONFIG folder, $GWFILE and $LAGACY_GWFILE
```

Comment:
nit: (could be subjective) I would not introduce these variables, they distract the reader from the only variable that this IPIP aims to define, which is `IPFS_GATEWAY`. 

```suggestion
### Configuration file with a list of preferred IPFS Gateways
```

If you want to highlight a term, perhaps remove `$CONFIG`, `$GWFILE` and `$LEGACY_GWFILE`, replace them with nouns in bold:

`$GWFILE` → **`gateways` list**
`$LEGACY_GWFILE` → **Kubo's `$IPFS_PATH/gateway` file**